spec/integration/denv_spec.rb

```diff
@@ -225,6 +225,28 @@
       end
     end
 
+    context 'repeat files' do
+      let(:env_path) { '../envs/c/.env' }
+      let(:local_path) { '../envs/c/.local.env' }
+      context 'separately' do
+        let(:first_merged_changes) { new_env.merge 'C' => '2' }
+        let(:second_merged_changes) { new_env.merge 'C' => '1' }
+        it 'respects the files and order' do
+          DEnv.from_file(env_path).from_file!(local_path)
+          expect(ENV.to_hash).to eq(first_merged_changes)
+          DEnv.from_file!(env_path)
+          expect(ENV.to_hash).to eq(second_merged_changes)
+        end
+      end
+      context 'together' do
+        let(:exp_changes) { new_env.merge 'C' => '1' }
+        it 'respects the files and order' do
+          DEnv.from_file(env_path).from_file(local_path).from_file!(env_path)
+          expect(ENV.to_hash).to eq(exp_changes)
+        end
+      end
+    end
+
```

Comment:
Hey @aemadrid, I believe the test is failing because there is no `c/.local.env`.  If you change `c` to `b`, `C` to `B`, then it will work.
```ruby
context 'repeat files' do
  let(:env_path) { '../envs/b/.env' } # b = 1
  let(:local_path) { '../envs/b/.local.env' } # b = 2
  context 'separately' do
    let(:first_merged_changes) { new_env.merge 'B' => '2' }
    let(:second_merged_changes) { new_env.merge 'B' => '1' }
    it 'respects the files and order' do
      DEnv.from_file(env_path).from_file!(local_path)
      expect(ENV.to_hash).to eq(first_merged_changes)
      DEnv.from_file!(env_path)
      expect(ENV.to_hash).to eq(second_merged_changes)
    end
  end
  context 'together' do
    let(:exp_changes) { new_env.merge 'B' => '1' }
    it 'respects the files and order' do
      DEnv.from_file(env_path).from_file(local_path).from_file!(env_path)
      expect(ENV.to_hash).to eq(exp_changes)
    end
  end
end
```

Reply:
you're right, my local .gitignore stopped me from sending that file and I didn't realize it. got it up and now it should be working.